src/assets/sass/protocol/components/_breadcrumb.scss

```diff
@@ -0,0 +1,36 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../includes/lib';
+
+.mzp-c-breadcrumb {
+    background-color: #f3f3f3;
+    margin-bottom: 0;
+    padding: 8px 18px;
+
+    .mzp-c-breadcrumb-item {
+        display: inline;
+        font-weight: 400;
+
+        .mzp-c-breadcrumb-item-link {
+            text-decoration: none;
+        }
+
+        &:not(.mzp-c-breadcrumb-title) {
+            font-weight: bold;
+            &:before {
+                content: ">";
```

Comment:
We probably shouldn't use the ">" character because it isn't actually an arrow, it already has a meaning (greater than). We can find some other unicode arrow character or maybe an SVG icon.

We also need to accommodate RTL and reverse the direction.

Reply:
Ok, good call out